docs/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md

```diff
@@ -603,7 +606,9 @@ The following section provides the code to build the example as a WPF app.
 
      A simple window that contains a text box and a button appears in the **Design** view of MainWindow.xaml.
 
-8. Add a reference for <xref:System.Net.Http>.
+8. In **Solution Explorer**, right-click on **References** and select **Add Reference...**.
```

Comment:
```suggestion
8. In **Solution Explorer**, right-click on **References** and select **Add Reference**.
```

I remember that it was mentioned somewhere not to include those dots.

Reply:
Ok, it makes sense. Those dots are quite useless.

Reply:
Found it [here](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions).

>  If a dialog box label ends with a colon or an ellipsis, don't include that end punctuation in instructions.